api/portainer.go

```diff
@@ -1121,7 +1121,7 @@ const (
 	// APIVersion is the version number of the Portainer API
 	APIVersion = "2.0.0"
 	// DBVersion is the version number of the Portainer database
-	DBVersion = 25
+	DBVersion = 26
```

Comment:
Database version update is also not required as database migration is not required.